site2/docs/architecture-overview.md

```diff
@@ -0,0 +1,143 @@
+---
+
+id: concepts-architecture-overview
+
+title: Architecture overview
+
+sidebar_label: Concepts

```

Comment:
The left-hand navigation we have on the site now reflects the old Information Architecture. As new pages are completed, many will apply to the new architecture rather than the old, so they may not fit neatly into one of the existing categories.

@Anonymitaet Let's discuss different possibilities for solving this problem in our subteam meeting this week.